backend/routers/detection.py

```
@@ -436,3 +439,49 @@ async def detect_abandoned_vehicle_endpoint(request: Request, image: UploadFile
+    _, image_bytes = await process_uploaded_image(image)
+
+    try:
+        client = get_http_client(request)
+        detections = await detect_cleanliness_clip(image_bytes, client=client)
+        return {"detections": detections}
+    except Exception as e:
+        logger.error(f"Cleanliness detection error: {e}", exc_info=True)
+        raise HTTPException(status_code=500, detail="Internal server error")
+
+@router.post("/api/detect-noise-pollution")
+async def detect_noise_pollution_endpoint(request: Request, file: UploadFile = File(...)):
```

Ruff (0.15.0)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)